test_utils/setup_tests/test_app_setup.py

```diff
@@ -0,0 +1,10 @@
+import os
```

Comment:
Again, I think we need to change the structure. This file is a test, not a test util (a test util in my mind is a helper function, not an actual test), so we should put this file in /tests and have /test_utils be within that folder